src/features/home/pages/FormPage.tsx

```diff
@@ -0,0 +1,11 @@
+import { FormRepeaterProvider } from '../../../context/FormRepeaterContext';
+import FormRepeater from '../FormRepeater/FormRepeater';
+
+const FormPage = () => {
+  return (
+    <FormRepeaterProvider>
+      <FormRepeater />
```

Comment:
This component can be completely written here, it does not have any reuse functionality that would take up a lot of space.

P.S. No need to overdo code split
but I also hope you find this article interesting https://reactjs.org/docs/code-splitting.html